command/ibft/ibft_switch.go

```diff
@@ -0,0 +1,247 @@
+package ibft
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/0xPolygon/polygon-edge/chain"
+	"github.com/0xPolygon/polygon-edge/command/helper"
+	"github.com/0xPolygon/polygon-edge/consensus/ibft"
+	"github.com/0xPolygon/polygon-edge/helper/common"
+	"github.com/0xPolygon/polygon-edge/types"
+)
+
+// IBFTSwitchCommand is the command to switch consensus
+type IBFTSwitchCommand struct {
+	helper.Base
+	Formatter *helper.FormatterFlag
+}
+
+func (c *IBFTSwitchCommand) DefineFlags() {
+	c.Base.DefineFlags(c.Formatter)
+
+	c.FlagMap["chain"] = helper.FlagDescriptor{
+		Description: fmt.Sprintf("Specifies the genesis file to update. Default: %s", helper.DefaultConfig().Chain),
+		Arguments: []string{
+			"GENESIS_FILE",
+		},
+		FlagOptional: true,
+	}
+
+	c.FlagMap["type"] = helper.FlagDescriptor{
+		Description: "Sets the new IBFT type [PoA, PoS]",
+		Arguments: []string{
+			"TYPE",
+		},
+		FlagOptional:      false,
+		ArgumentsOptional: false,
+	}
+
+	c.FlagMap["deployment"] = helper.FlagDescriptor{
+		Description: "Sets the height to deploy the contract in PoS",
+		Arguments: []string{
+			"DEPLOYMENT",
+		},
+		FlagOptional:      true,
+		ArgumentsOptional: false,
+	}
+
+	c.FlagMap["from"] = helper.FlagDescriptor{
+		Description: "Sets the height to switch the new type",
+		Arguments: []string{
+			"FROM",
+		},
+		FlagOptional:      false,
+		ArgumentsOptional: false,
+	}
+}
+
+// GetHelperText returns a simple description of the command
+func (c *IBFTSwitchCommand) GetHelperText() string {
+	return "Add settings in genesis.json to switch IBFT type"
+}
+
+func (c *IBFTSwitchCommand) GetBaseCommand() string {
+	return "ibft switch"
+}
+
+// Help implements the cli.PeersAdd interface
```

Comment:
Leftover comment from copy paste 🙂

Reply:
Please replace all references to `cli.PeersAdd` in this file 🙏

Reply:
Fixed at 4dd5210951014804bcc12ecbb507d80374ba1121